src/assets/TokenListController.ts

```diff
@@ -166,9 +181,39 @@ export class TokenListController extends BaseController<
   }
 
   /**
-   * Fetching token list from the Token Service API
+   * Fetching token list
    */
   async fetchTokenList(): Promise<void> {
+    if (this.useStaticTokenList) {
+      await this.fetchFromStaticTokenList();
+    } else {
+      await this.fetchFromDynamicTokenList();
+    }
+  }
+
+  /**
+   * Fetching token list from the contract-metadata as a fallback
+   */
+  async fetchFromStaticTokenList(): Promise<void> {
+    const tokenList: TokenMap = {};
+    for (const tokenAddress in contractmap) {
+      const { erc20, logo, ...token } = contractmap[tokenAddress];
+      if (erc20) {
```

Comment:
Is there a reason we don't want to show the ERC721 tokens?

Reply:
This token list will be used for the detection of ERC20 tokens. The ERC721 tokens are handles separately as `collectibles` in this repo.